test/unit_test.c

```diff
@@ -134,21 +134,26 @@ static void test_commalist(void) {
   struct mg_str s4 = mg_str("a=123"), s5 = mg_str("a,b=123");
   ASSERT(mg_commalist(&s1, &k, &v) == false);
 
+  v.len = k.len = 42;
   ASSERT(mg_commalist(&s2, &k, &v) == true);
   ASSERT(v.len == 0 && mg_vcmp(&k, "a") == 0);
   ASSERT(mg_commalist(&s2, &k, &v) == false);
 
+  v.len = k.len = 42;
   ASSERT(mg_commalist(&s3, &k, &v) == true);
   ASSERT(v.len == 0 && mg_vcmp(&k, "a") == 0);
+  v.len = k.len = 42;
   ASSERT(mg_commalist(&s3, &k, &v) == true);
   ASSERT(v.len == 0 && mg_vcmp(&k, "b") == 0);
   ASSERT(mg_commalist(&s3, &k, &v) == false);
 
+  v.len = k.len = 42;
   ASSERT(mg_commalist(&s4, &k, &v) == true);
   ASSERT(mg_vcmp(&k, "a") == 0 && mg_vcmp(&v, "123") == 0);
   ASSERT(mg_commalist(&s4, &k, &v) == false);
   ASSERT(mg_commalist(&s4, &k, &v) == false);
 
+  v.len = k.len = 42;
```

Comment:
Is this magic number explained somewhere ?

Reply:
No, it is just a random number (the famous answer), just to make sure we're not re-using the result of the previous test.